packages/webview_flutter/webview_flutter_android/lib/webview_surface_android.dart

```diff
@@ -59,7 +53,8 @@ class SurfaceAndroidWebView extends AndroidWebView {
               // WebView content is not affected by the Android view's layout direction,
               // we explicitly set it here so that the widget doesn't require an ambient
               // directionality.
-              layoutDirection: TextDirection.rtl,
```

Comment:
This shouldn't be a breaking change for large majority of people. This value is ignored unless support for this is explicitly added to the `AndroidManifest.xml`. For example, it is still `ltr` in our example app.